pkg/console/util.go

```diff
@@ -325,25 +334,44 @@ func execute(ctx context.Context, g *gocui.Gui, env []string, cmdName string) er
 	if err != nil {
 		return err
 	}
+	defer stderr.Close()
+
 	stdout, err := cmd.StdoutPipe()
 	if err != nil {
 		return err
 	}
+	defer stdout.Close()
+
+	var wg sync.WaitGroup
+	var writeLock sync.Mutex
+
+	wg.Add(2)
+	go func() {
+		defer wg.Done()
+		printToPanelAndLog(g, installPanel, "[stderr]", stderr, &writeLock)
+	}()
+
+	go func() {
+		defer wg.Done()
+		printToPanelAndLog(g, installPanel, "[stdout]", stdout, &writeLock)
+	}()
+
 	if err := cmd.Start(); err != nil {
 		return err
 	}
 
-	scanner := bufio.NewScanner(stdout)
-	scanner.Split(bufio.ScanLines)
-	for scanner.Scan() {
-		printToPanel(g, scanner.Text(), installPanel)
-	}
-	scanner = bufio.NewScanner(stderr)
-	scanner.Split(bufio.ScanLines)
+	wg.Wait()
+	return cmd.Wait()
+}
+
+func printToPanelAndLog(g *gocui.Gui, panel string, logPrefix string, reader io.Reader, lock *sync.Mutex) {
+	scanner := bufio.NewScanner(reader)
 	for scanner.Scan() {
-		printToPanel(g, scanner.Text(), installPanel)
+		logrus.Infof("%s: %s", logPrefix, scanner.Text())
```

Comment:
For reviewer: the default splitfunc is `ScanLines`, so we are happy to remove unnecessary `scanner.Split` calls.

https://cs.opensource.google/go/go/+/refs/tags/go1.16.5:src/bufio/scan.go;l=88-94;drc=refs%2Ftags%2Fgo1.17.7